applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -325,6 +317,20 @@ static void button_evt_handler(struct button_evt event)
 
 		break;
 
+	case BUTTON_5:
+#if (CONFIG_TRANSPORT_BIS) && (CONFIG_AUDIO_DEV == HEADSET) && !(AUDIO_TEST_MUTE)
```

Comment:
Should AUDIO_TEST_MUTE be renamed to MUTE_BTN_ENABLED?

Reply:
Should this be CONFIG_AUDIO_MUTE?
Can we just test against CONFIG_AUDIO_MUTE as with the tone test?

Reply:
Changed to what Graham said

Reply:
That is better, but IMO this sounds like we enable muting - which is sort of true.. 
What about CONFIG_BTN_AUDIO_MUTE or CONFIG_MUTE_USING_BUTTON?

Reply:
This is why it had the "TEST" in the name

Reply:
I agree that this is not the best name.
But I dont think it is that much better to include "button" in the name, since the only way we can mute now is by using a button, so it is practically the same.
Ideally it should be something like "CONFIG_AUDIO_MUTE_ON_BIS_HEADSET_INSTEAD_OF_CHANGING_GATEWY_DEVICE", but that is of course too long